src/resolvers/create-todo.resolver.ts

```diff
@@ -0,0 +1,16 @@
+import { Inject } from '@nestjs/common';
+import { Args, Mutation, Resolver } from '@nestjs/graphql';
+import { Todo } from '@/database/entities/todo.entity';
+import { CreateTodoInput } from '@/inputs/create-todo.input';
+import { CreateTodoService as ICreateTodoService } from '@/interfaces/create-todo.interface';
+import { CreateTodoService } from '@/services/create-todo.service';
+@Resolver(() => Todo)
+export class CreateTodoResolver {
+  constructor(
+    @Inject(CreateTodoService) private createTodoService: ICreateTodoService,
```

Comment:
Is really necessary to use "ICreateTodoService"? Why not "CreateTodoService"?

Reply:
Yes, Im applying dependency inversion principle on high level modules